citi-onboarding/Recorda-one-page
# feat(footer-CRUD): created the footer database

### **What** I did:
- Created the Footer model
- Created the FooterController
- Created the CRUD routes to the FooterController
- Deleted the User model, UserController and the CRUD routes to the UserController

### **How to test**
- git checkout main
- git pull
- git checkout feature/footer-CRUD
- git pull
- open VSCode terminal
- write "docker compose up"
- open Insomnia and Postbird
- test with "get", "put", "post" and "delete" at Insomnia
- check if it's all working on Postbird
